frameit/lib/frameit/template_finder.rb

```diff
@@ -11,32 +10,36 @@ class TemplateFinder
     def self.get_template(screenshot)
       return nil if screenshot.mac?
 
-      filename = "Apple #{screenshot.device_name} #{screenshot.color}"
-
-      templates = Dir["#{FrameDownloader.templates_path}/#{filename}.{png,jpg}"] # ~/.frameit folder
+      filename = create_file_name(screenshot.device_name, screenshot.color.nil? ? screenshot.default_color : screenshot.color)
+      templates = Dir["#{FrameDownloader.templates_path}/#{filename}.{png,jpg}"] # ~/.frameit_android folder
```

Comment:
`~/.frameit_android` why is this one Android specific?

Reply:
I forgot to revert this change. In the beginning I was making the frameit with Android as a separate module. Will revert in a new commit.

Reply:
[Fixed in the new commit.](https://github.com/milhauscz/fastlane/commit/e52dafa0a43f9306e0ad8245e81ceac5a0550065)